pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -61,8 +61,30 @@ const (
 
 	operationPollInterval        = 3 * time.Second
 	operationPollTimeoutDuration = 30 * time.Minute
+
+	defaultLBSourceRange = "0.0.0.0/0"
+
+	//Expected annotations for GCE
+	gceLBAllowSourceRange = "net.beta.kubernetes.io/gce-source-ranges"
```

Comment:
dont' we want to make it explicit that this is cloudprovider specific? (by putting gce- in there)